addons/mail/static/src/models/messaging.js

```diff
@@ -131,18 +136,19 @@ registerPatch({
          *
          * If a chat is not appropriate, a notification is displayed instead.
          *
-         * @param {Object} person forwarded to @see `getChat()`
-         * @param {Object} [options] forwarded to @see `Thread:open()`
+         * @param {Object} person forwarded to @see `Messaging.syncPersona()`
```

Comment:
I suppose `Persona/sync` to refer to?

Reply:
Oops, thanks for pointing that out, I was referring to `Messaging/getSyncedPersona` :+1: